integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDynamicUpdate.java

```diff
@@ -922,8 +929,8 @@ public void testOnNonDynamicChangesCancelUpdate() {
 
     // check that the domain status condition type is "OnlineUpdateCanceled" and message contains the expected msg
     String expectedMsgForCancelUpdate = "Online update completed successfully, but the changes require restart and "
-          + "the domain resource specified 'spec.configuration.model.onlineUpdate.onNonDynamicChanges=CancelUpdate' "
-          + "option to cancel all changes if restart require.";
+        + "the domain resource specified 'spec.configuration.model.onlineUpdate.onNonDynamicChanges=CancelUpdate' "
+        + "option to cancel all changes if restart require.";
```

Comment:
We don't have cancelUpdate anymore.  Do we still need this?

Reply:
Currently the test is disabled. If this feature is not going to be added in future releases I can delete this test. Please confirm.

Reply:
It's unlikely we will add this back soon, and if we indeed put it back then you can always use history to get the code back.

Reply:
ok, removed the test